exporting/send_data.c

```diff
@@ -53,7 +57,39 @@ void simple_connector_receive_response(int *sock, struct instance *instance)
         buffer_need_bytes(response, 4096);
 
         ssize_t r;
+#ifdef ENABLE_HTTPS
+        if (connector_specific_config->conn && connector_specific_config->flags == NETDATA_SSL_HANDSHAKE_COMPLETE) {
+            r = (ssize_t)SSL_read(connector_specific_config->conn,
+                                  &response->buffer[response->len],
+                             (int) (response->size - response->len));
```

Comment:
Align it please.

Reply:
At Clion it is aligned, I will take a deeper look on this.

Reply:
I also fixed this alignment.